frontend/public/components/utils/owners-resource-link.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { K8sResourceKind, referenceForOwnerRef } from '../../module/k8s';
+import { ResourceLink } from './index';
+
+export const OwnersResourceLink: React.FC<OwnersResourceLinkProps> = ({resource}) => {
+  const owners = (_.get(resource.metadata, 'ownerReferences') || [])
+    .map((o, i) => <ResourceLink key={i} kind={referenceForOwnerRef(o)} name={o.name} namespace={resource.metadata.namespace} title={o.uid} />);
```

Comment:
Owner UID is a better key. Use the `OwnerReference` type

```suggestion
    .map((o: OwnerReference[]) => <ResourceLink key={o.uid} kind={referenceForOwnerRef(o)} name={o.name} namespace={resource.metadata.namespace} />);
```

https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318